libs/evcs-migration/src/main/java/uk/gov/di/ipv/core/library/persistence/item/ReportUserIdentityItem.java

```diff
@@ -41,6 +41,11 @@ public String getHashUserId() {
         return hashUserId;
     }
 
+    @JsonIgnore
+    public String getUserId() {
+        return userId;
+    }
+
```

Comment:
Do we need this? We've already got a getter through the lombok `@Data` annotation, and we have `@JsonIgnore` on the field, which is excluding the value from serialisation/deserialisation.